main.go

```diff
@@ -28,34 +30,42 @@ var (
 )
 
 func init() {
+	zerolog.TimeFieldFormat = zerolog.TimeFormatUnix
 
 	flag.StringVar(&Token, "t", "", "Bot Token")
 	flag.Parse()
 }
 
 func main() {
+	log.Info().
+		Str("Redis address", *redisAddr).
+		Msg("Discord starting up!")
 
 	rdb = rdbConnect(*redisAddr)
 	ctx = context.Background()
 
 	// Create a new Discord session using the provided bot token.
 	dg, err := discordgo.New("Bot " + Token)
 	if err != nil {
-		fmt.Println("error creating Discord session,", err)
-		return
+		log.Error().
+			Str("error creating Discord session,", err.Error()).
```

Comment:
You could also use zerolog's integrated error logging, like this:
`.Err(err)` and have a more useful `.Msg` content ?